sdrbase/util/crc.h

```diff
@@ -105,6 +114,8 @@ class SDRBASE_API crc32 : public crc
     crc32() : crc(32, 0x04C11DB7, false, 0xffffffff, 0xffffffff) {}
 };
 
+// Should probably try to use SSE 4.2's CRC32C instruction if available
+// See _mm_crc32_u8 and _mm_crc32_u64 intrinsics
```

Comment:
Problem is that it should also compile for ARM

Reply:
Sure. It doesn't currently use them, and if it did, it would be implemented so that there would be an alternative if not available. 
Some versions of ARM processors also have crc32c insn, but not all.

Reply:
BTW when I say "ARM" I mean armv8 a.k.a. arm64. I do not aim for support of earlier versions. Practically this addresses RPi 3B+ and RPi 4